services/webhook/webhook.go

```diff
@@ -95,11 +95,16 @@ func getPayloadBranch(p api.Payloader) string {
 }
 
 // PrepareWebhook adds special webhook to task queue for given payload.
-func PrepareWebhook(w *models.Webhook, repo *models.Repository, event models.HookEventType, p api.Payloader) error {
-	if err := prepareWebhook(w, repo, event, p); err != nil {
+func PrepareWebhook(w *models.Webhook, repo *models.Repository, org *models.User, event models.HookEventType, p api.Payloader) error {
```

Comment:
It probably would be simpler to have:

```go
func PrepareWebhook(w *models.Webhook, repo *models.Repository, event models.HookEventType, p api.Payloader) error {
  return PrepareRepoOrgWebhook(w, repo, nil, event, p)
}

func PrepareOrgWebhook(w *models.Webhook, org *models.User, event models.HookEventType, p api.Payloader) error {
  return PrepareRepoOrgWebhook(w, nil, org, event, p)
}

func PrepareRepoOrgWebhook(w *models.Webhook, repo *models.Repository, org *models.User, event models.HookEventType, p api.Payloader) error {
 ...
}
```

Then you would need most of the changes in modules/notification/webhook/webhook.go and elsewhere.